OpenRA.Mods.Common/Traits/SupportPowers/AirstrikePower.cs

```diff
@@ -36,6 +38,12 @@ public class AirstrikePowerInfo : SupportPowerInfo
 		[Desc("Amount of time to keep the camera alive after the aircraft have finished attacking")]
 		public readonly int CameraRemoveDelay = 25;
 
+		[Desc("Placeholder cursor animation for the target cursor when the real cursor is invisible.")]
+		public readonly string TargetPlaceHolderCursorAnimation = null;
+
+		[Desc("Directional cursor used to draw the arrows.")]
+		public readonly string DirectionCursorAnimation = null;
```

Comment:
These don't necessarily need to be cursors, so how about `DirectionArrowAnimation`?